smart-contracts/scripts/helpers/utils.js

```diff
@@ -93,10 +108,37 @@ function cacheBuster(url) {
   return finalUrl;
 }
 
+/**
+ * Removes duplicates from arrays
+ * @param {array} list Your array
+ * @returns {array} an array containing no duplicates
+ */
+function removeDuplicates(list) {
+  const uniqueSet = new Set(list);
+  return [...uniqueSet];
+}
+
+/**
+ * Compares two arrays and returns true if they have the exact same elements, even if out of order
```

Comment:
Should mention in the comment that multiple of the same elements are not ignored so if I have an array [1, 1, 2, 3] and another [1, 2, 3] though they have the same elements this will report false.